.lsp/config.edn

```diff
@@ -1,4 +1,8 @@
 {:keep-require-at-start?        true
  :show-docs-arity-on-same-line? true
  :project-specs                 [{:project-path "deps.edn"
-                                  :classpath-cmd ["clojure" "-A:dev:ee:ee-dev:drivers:drivers-dev" "-Spath"]}]}
+                                  :classpath-cmd ["clojure" "-A:dev:ee:ee-dev:drivers:drivers-dev" "-Spath"]}]
+
+ :linters {:clojure-lsp/unused-public-var {:level :warning
+                                           :exclude-when-defined-by
+                                           #{metabase.api.common/defendpoint}}}}
```

Comment:
this seems to not work right now which is a shame

Reply:
You could just write a custom macroexpansion for `defendpoint` that Kondo will use to lint it

Reply:
yeah @borkdude will be doing that soon i think

Reply:
Will do on Thursday!